README.md

```diff
@@ -67,7 +67,8 @@ void main() {
 
 Nyxx examples can be found [here](https://github.com/l7ssha/nyxx/tree/dev/nyxx/example).
 
-Commander examples can be found [here](https://github.com/l7ssha/nyxx/tree/dev/nyxx.commander/example)
+Commander examples can be found [here](https://github.com/l7ssha/running_on_dart)
```

Comment:
Should stay as is, because that another bot just written on framework. There no examples yet so link is invalid, but will add them before release.

Reply:
Okay, updated.